Packages/com.unity.inputsystem/InputSystem/Devices/Keyboard.cs

```diff
@@ -2009,21 +2077,33 @@ public KeyControl this[Key key]
         public static Keyboard current { get; private set; }
 
         /// <summary>
-        /// Make the keyboard the current keyboard (i.e. <see cref="current"/>).
+        /// Make a keyboard the current active keyboard.
```

Comment:
Could we do inheritDoc on this since defined in base? It should not change meaning in base classes right?

Reply:
I think it's neat to have a example code for this, but I added the inherited doc above for more information: [58eeef6](https://github.com/Unity-Technologies/InputSystem/pull/2069/commits/58eeef63ae15b8a2e4fd38e838159d003ab9bc3c)